Avo.ts

```diff
@@ -1029,13 +1034,7 @@ let SegmentInspectorForDocs = {
   /* Segment does not have revenue api, @see https://segment.com/docs/spec/track/#properties */
   logEvent: function logEvent(eventName: any, eventProperties: any, integrations: any) {
     if (typeof (window as any) === 'undefined') { return; }
-    try {
-      (this as any).getInstance().track(eventName, eventProperties, Object.assign({}, {integrations: integrations}));
-    } catch(e) {
-      if(console && console.error) { 
-        console.error('Segment track error', e);
-      }
-    }
```

Comment:
Maybe we want to keep this try-catch until we've verified it ain't a problem any more?